packages/components/src/components/tooltip/tooltip.stories.ts

```diff
@@ -65,198 +70,81 @@ export const Default = {
-        }
-      },
-      args,
-      constants: [
-        { type: 'template', name: 'width', value: '<div class="template-position">%TEMPLATE%</div>' },
-        {
-          type: 'attribute',
-          name: 'placement',
-          value: 'bottom'
-        }
-      ]
-    })
+/**
+ * Use the `placement` attribute to set the placement.
+ *
+ * - `top-start`: displays tooltip above the trigger element and aligns arrow to the start of the tooltip container
+ * - `top`: displays tooltip above the trigger element
+ * - `top-end`: displays tooltip above the trigger element and aligns arrow to the end of the tooltip container
+ * - `bottom-start`: displays tooltip below the trigger element and aligns arrow to the start of the tooltip container
+ * - `bottom`: displays tooltip below the trigger element
+ * - `bottom-end`: displays tooltip below the trigger element and aligns arrow to the end of the tooltip container
+ */
+export const Placement = {
+  render: () => html`
+    <div class="grid grid-cols-3 gap-12">
+      <div>
+        <sd-tooltip content="Lorem ipsum" placement="top-start" open size="lg" trigger="click focus"></sd-tooltip>
+      </div>
+      <div>
+        <sd-tooltip content="Lorem ipsum" placement="top" open size="lg" trigger="click focus"></sd-tooltip>
+      </div>
+      <div>
+        <sd-tooltip content="Lorem ipsum" placement="top-end" open size="lg" trigger="click focus"></sd-tooltip>
+      </div>
+      <div>
+        <sd-tooltip content="Lorem ipsum" placement="bottom-start" open size="lg" trigger="click focus"></sd-tooltip>
+      </div>
+      <div>
+        <sd-tooltip content="Lorem ipsum" placement="bottom" open size="lg" trigger="click focus"></sd-tooltip>
+      </div>
+      <div>
+        <sd-tooltip content="Lorem ipsum" placement="bottom-end" open size="lg" trigger="click focus"></sd-tooltip>
+      </div>
+    </div>
+  `
 };
 
+/**
+ * Use the `disabled` attribute to disable the tooltip trigger.
+ */
 export const Disabled = {
-  parameters: { controls: { exclude: 'disabled' } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'disabled'
-        }
-      },
-      args,
-      constants: [
-        { type: 'template', name: 'width', value: '<div class="template-position">%TEMPLATE%</div>' },
-        {
-          type: 'attribute',
-          name: 'placement',
-          value: 'bottom'
-        }
-      ]
-    })
+  render: () => html` <sd-tooltip content="Lorem ipsum" placement="top" size="lg" disabled></sd-tooltip> `
 };
 
 /**
- * In case of having the content longer than the space available, the tooltip can be set to scrollable by overriding the `pointer-events` property to auto. In this case the padding-right should also be adjusted via `sd-tooltip::part(body)`. This behavior is heavily discouraged, but it is available in case of need.
+ * Use the `trigger` attribute to control how the tooltip is triggered.
```

Comment:
For the tigger sample: can we use a "top-start instead"?
So that when we click the trigger has some margin to the left side.

Reply:
Updated